makeWorkspace/make_ws.py

```diff
@@ -273,6 +274,38 @@ def get_stat_variations(obj, category):
     histograms[name_dn] = h_dn
   return histograms
 
+def get_mergedMC_stat_variations(to_merge_mc_bkgs, category):
+  histograms = {}
+  for region_name in to_merge_mc_bkgs.keys():
+    merged_name = str(region_name) + "_mergedMCBkg"
+    merged_obj = to_merge_mc_bkgs[region_name][0].Clone(merged_name)
+    if len(to_merge_mc_bkgs[region_name])>1:
+      for obj in to_merge_mc_bkgs[region_name][1:]:
+          merged_obj = merged_obj + obj
+    for ibin in range(1,merged_obj.GetNbinsX()+1):
+      variation_name_up =  "{NAME}_{category}_stat_bin{ibin}Up".format(category=category,NAME=merged_name, ibin=ibin)
+      variation_name_dn =  "{NAME}_{category}_stat_bin{ibin}Down".format(category=category,NAME=merged_name, ibin=ibin)
+      merged_central = merged_obj.GetBinContent(ibin)
+      merged_error = merged_obj.GetBinError(ibin)
+      if merged_central<=0:
```

Comment:
Here you continue if the central value is 0. In this case, we must also remove the corresponding nuisance from the data card. Is that already covered by the existing functionality in make_cards?

Reply:
yeah I think it's covered by that